test/run_test.py

```diff
@@ -1054,6 +1102,33 @@ def export_S3_test_times(test_times_filename: str, test_times: Dict[str, float])
         file.write('\n')
 
 
+def load_specified_test_cases(filename: str) -> None:
+    if not os.path.exists(filename):
+        print(f'Could not find specified tests file: {filename}. Proceeding with default behavior.')
+        return
+
+    import csv
+    with open(filename, mode='r', encoding="utf-8-sig") as csv_file:
```

Comment:
Why is this `utf-8-sig` instead of just `utf-8`?

Reply:
There is this byte-order-mark symbol that doesn't get decoded when I just use utf-8, but using utf-8-sig works.

Reply:
whoa mysterious

Reply:
but ok

Reply:
could you write that in a code comment?

Reply:
sure